DAQ/src/StrawRecoFromArtdaqFragments_module.cc

```diff
@@ -0,0 +1,297 @@
+// ======================================================================
+//
+// StrawRecoFromArtdaqFragments:  add tracker data products to the event
+//
+// ======================================================================
+
+#include "art/Framework/Core/EDProducer.h"
+#include "art/Framework/Principal/Event.h"
+#include "art/Framework/Services/Registry/ServiceHandle.h"
+#include "fhiclcpp/ParameterSet.h"
+
+#include "art/Framework/Principal/Handle.h"
+#include "artdaq-core-mu2e/Data/TrackerDataDecoder.hh"
+#include "artdaq-core-mu2e/Overlays/FragmentType.hh"
+
+#include "Offline/DataProducts/inc/TrkTypes.hh"
+#include "Offline/RecoDataProducts/inc/IntensityInfoTrackerHits.hh"
+#include "Offline/RecoDataProducts/inc/ProtonBunchTime.hh"
+#include "Offline/RecoDataProducts/inc/StrawDigi.hh"
+
+#include <artdaq-core/Data/Fragment.hh>
+#include <artdaq-core/Data/ContainerFragment.hh>
+
+#include <iostream>
+
+#include <string>
+
+#include <memory>
+
+#define TRACEMF_USE_VERBATIM 1
+
+#include "TRACE/tracemf.h"
+#define TRACE_NAME "StrawRecoFromArtdaqFragments"
+
+namespace art {
+  class StrawRecoFromArtdaqFragments;
+}
+// ======================================================================
+
+class art::StrawRecoFromArtdaqFragments : public EDProducer {
```

Comment:
My preference would be to call this "StrawDigiFrom" instead of "StrawRecoFrom" since there is no reco in the way I'd use the word.